CODEOWNERS

```diff
@@ -11,7 +11,9 @@
 # Search
 /sql/moz-fx-data-shared-prod/search_terms @whd @jasonthomas
 /sql/moz-fx-data-shared-prod/search_terms_derived @whd @jasonthomas
+/sql/moz-fx-data-shared-prod/search_dervied/search_revenue_levers_daily_v1 @mozilla/revenue_forecasting_data_reviewers
```

Comment:
Is this error is because the group doesn't have permissions yet to the repository? If so, then it can be solved by inheriting from the telemetry team.